web/src/OverviewResourceBar.tsx

```diff
@@ -1,7 +1,16 @@
-import React from "react"
+import React, { Component, useRef, useState } from "react"
 import styled from "styled-components"
+import { AccountMenuContent, AccountMenuHeader } from "./AccountMenu"
+import { incr } from "./analytics"
+import { ReactComponent as AccountIcon } from "./assets/svg/account.svg"
+import { ReactComponent as HelpIcon } from "./assets/svg/help.svg"
+import { ReactComponent as StatusSquareIcon } from "./assets/svg/status-square.svg"
```

Comment:
SVG is fine, but for a simple design like this it can also just be a div with fixed width / height / border radius in CSS.

Reply:
that's fair! changed it to pure html